ls1intum/Artemis
# `Programming exercises`: Show participation buttons for inactive exercises

### Checklist
#### General


#### Client

### Motivation and Context



https://github.com/ls1intum/Artemis/issues/4614

### Description


Display code editor buttons if Programming Exercise is in INACTIVE state, too. Previosuly, it was only display in INITIALIZED state.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Prorgramming exercise the student participated in and the build plan has already been cleaned up

1. Open the exercise view
2. Verify that the buttons to open the code editor or clone the repo exist

### Review Progress





#### Code Review
#### Manual Tests
